README.md

```diff
@@ -20,9 +22,9 @@ goal means that it doesn't have features such as tabs or splits (which can be
 better provided by a window manager or [terminal multiplexer][tmux]) nor
 niceties like a GUI config editor.
 
-The software is considered to be at an **alpha** level of readiness--there are
-missing features and bugs to be fixed, but it is already used by many as a daily
-driver.
+The software is considered to be at a **beta** level of readiness -- there are
+a few missing features and bugs to be fixed, but it is already used by many as
+a daily driver.
```

Comment:
While bumping Alacritty's version, I noticed that it's also titled as 'alpha' in the snap file. Should probably be bumped there too.

Might be worth searching through the other files too.

Reply:
Is there any sane way to DRY up all this? I'm fine making manual changes to the `man` entry since it's formatting is a bit of a special case, but I feel like package descriptions like this should be copy pastes (essentially) from the README. Worst case, it's not too hard to just do that, but I've never dealt with a snapcraft file before.

Reply:
I'm completely unfamiliar with snapcraft and don't even know if this file is actually used by the snap package, since no Alacritty maintainer also maintains the snapcraft file.

It might be interesting to get a hold on whoever maintains that and put that effort on them? I feel like this is something that should be done by the packager.

If we do keep it in the repo, I don't think there's any good way around it though. Though it might be possible to automatically generate the manpage from the CLI --help and readme.